frontend/src/pages/customize/index.tsx

```diff
@@ -0,0 +1,161 @@
+
+  const customizeForm = useForm<IBusinessDetails>({
+    defaultValues: {
+      businessName: auth.user?.businessName || '',
+      businessDescription: auth.user?.businessDescription || '',
+    },
+    onSubmit: async ({ value }) => {
+      try {
+        setIsLoading(true);
+
+        const tokens = await clientsService.updateClientInfo(value);
+
+        auth.setUser({
+          ...auth.user!,
+          ...value,
+        });
+
+      writeTokens(tokens, true);
+
+      flushSync(() => {
+        const payload = jwtDecode<JwtPayload & IUserDetails>(tokens.accessToken, {});
+        auth.setUser(payload);
+      });
+
+      } catch {
+        setErrorOccurred(true);
+      } finally {
+        setIsLoading(false);
+      }
+    },
+    validators: {
+      onSubmit({ value }) {
+        const requiredFields = ['businessName', 'businessDescription'] as Array<keyof typeof value>;
+        if(requiredFields.some((field) => isEmpty(value[field].toString())) || value.businessDescription.length > 200) {
+          setErrorOccurred(true);
+          return 'Missing or invalid values';
+        }
+    },
+  }
+  });
+
+  return (
+    <>
+      <Grid container justifyContent={'center'} spacing={4}>
+        <Grid item md={12}>
+          <Typography variant="h6" sx={{ fontWeight: 'regular' }}>
+            Customize your recommendations and settings
+          </Typography>
+        </Grid>
+        <Grid item md={9}>
+          <Paper
+            elevation={1}
+            sx={{
+              backgroundColor: 'white',
+              borderRadius: '1vh',
+              height: '60vh',
+            }}
+          >
+            <Stack direction={'column'} padding={8} spacing={2}>
+              <customizeForm.Field
+                name="businessName"
+                children={(field) => (
+                  <TextField
+                    fullWidth
+                    id="businessName-input"
+                    label="Business Name"
+                    type="string"
+                    variant="outlined"
+                    value={field.state.value}
+                    onBlur={field.handleBlur}
+                    onChange={(e) => field.handleChange(e.target.value)}
+                  />
+                )}
+              ></customizeForm.Field>
+              <customizeForm.Field
+                name="businessDescription"
+                children={(field) => (
+                  <TextField
+                    fullWidth
+                    id="businessDescription-input"
+                    label="Business Description"
+                    type="string"
+                    variant="outlined"
+                    value={field.state.value}
+                    onBlur={field.handleBlur}
+                    onChange={(e) => field.handleChange(e.target.value)}
+                    multiline
+                    rows={7}
+                    helperText="Up to 200 characters"
+                    error={field.state.value.length > 200}
```

Comment:
Does this block the user from sending the form?

Reply:
No, it just for showing red line around the text area once you typed over 200 letters, the actual validation and prevention is in the `validators` field - line 48.